src/Homepage/SearchResults.js

```diff
@@ -29,9 +39,10 @@ class SearchResults extends React.Component {
   }
 
   render() {
-    const selectedResult = this.props.results.find(result =>
+    const selectedResult = this.props.results.find(result => (
```

Comment:
i think you might have fixed https://github.com/covid-maps/covid-maps/issues/159

Reply:
i'm currently using the store name, should i switch to `storeId` ?

Reply:
Do you send the store data in the form submission api response ?

Reply:
Yes the submission response sends back store data. I think store name is ok for now